ios/brave-ios/Sources/Brave/Assets/SearchPlugins/yahoojp.xml

```diff
@@ -0,0 +1,20 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+<SearchPlugin xmlns="http://www.mozilla.org/2006/browser/search/">
+<ShortName>Yahoo! JAPAN</ShortName>
+<InputEncoding>UTF-8</InputEncoding>
+<Image width="64" height="64">data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEgAAABICAYAAABV7bNHAAAACXBIWXMAACE4AAAhOAFFljFgAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAARHSURBVHgB7ZyxUxNBFMbfoaWMtFpBY030DyD0olCqhaEQ7QClT+hVpFMbQuFYCkKfpBehtiE2tOJgYxX3y2aHy3m7b++yl4HwfjNHEnJJ7r59++3bt8lF1KPT6Uyom2W1lXvbVeRIbZtRFNXNPyL8UeJMqpuG2iZJAG21zSqh2kagYxJxkrTVVrqmxKmoOxUSksBy/kZKoEN1Z5qENJoQqEOClTESnIhADCIQgwjEIAIxiEAMIhCDCMQgAjGIQAwiEIMIxCACMYhADNfZPRbWiE7PKCiTt4m2qnQZ4AXaaVJw5sfpsuAumCFy6vtEP09Uvf+HfoxbX+bLKlpuqeKlEuTmuL6duKHWTO7p+5eAfBVFiNQ8IFp9496v8UGL4QsaIFR3nhg/b4TSY1V1nyOaUccyfYeyMFjJtb5HtLhufx5ec/jJP1ogPE4mBMdf9efvNlUkr/Uf0/wM0cOyV+MNNoqhVapL9ufbqmuuviVvMrauFRwThAA7LfrvmN59Vqtez4mmHugGdkQtb9IcNXUwv8/0h6aBKMOJLz8iLzrf+v3O+B+6dBqITrw/NtOFjDjA9joAscjth+FWNdAirpNAV4sfeBY2lfgrCb8zqYKrm/h0WdMVLYRLFL+8tn8QoiFvPoWTTIqD0RGCcx4C/3ER74oWwgmEKGm8t4crTnT9I2UCr0Fkxtl4pRvDx/hdORyEqdwnjrBTDXwoDt4GfGp7n7zAfugeJurw3kgbVjy9rH3iztk8ogeEn4sh7F0jG7pL1xwdINIqtfPHMF5EZ5acqvnd/pxn9IBiJqsY2eATaXB+hOSzFuuKT+d05GQ1eJf/uKI8QXGzeYwwtpNK8yMIBr+JpwuIxHo137TENqIid8uQbxUnEE7KZaZxP0KXKz3pPymIU1uiXOB9bBFafUZZKLYehJbi/AgnM/ui35e2avnFAbaBwNOY4wzn6y/wHN+yCcTxNFArmEIkBwIIg6QwI8OpKLr8KA5ynEHFgb+ljZLVfBE5HIHgQxsv3fvAPH1zHBetFHPOMKwnGV5Nmhs5UFgLQVpXRg6Vk9Eq2qNrJYd3RGbeSTKNmkBp2XPGYT3JaAmEenc8g88xrCcZvGB2kYA4RiB4kW26k4HRXTgMIA4YnkCnf6gwEC1chSAnw+tirQMqBIxaC71VC9PFZu4O7D2GYqcapmi129LFexfIk8yColloNLmRqw6EVYnke2N/lEgCEF4glDFqGUurLiDWr0b6c2gAzLuSoIpwYT0o+BcdHBk2fC2ZoXcXBssUivARlFw+ji8BF0U3g1ZJ4vaerkAOOuGNIb/2YZAvUDGIQAwiEIMIxCACMYhADCIQgwjEIAIxiEAMIhCDCMQgAjGIQAwiEAMEapJg4wgCtUiwsRn1Lu6GiyxNkhCnHUXR1Jj6c6oezJK+ZpegaZPWRJt092pvSi11d5H0pfKuKk214fddJWiCf/wDlQxkzCr1BJgAAAAASUVORK5CYII=</Image>
+<Url type="application/x-suggestions+json" method="GET" template="https://search.yahooapis.jp/AssistSearchService/V2/webassistSearch">
+  <Param name="p" value="{searchTerms}"/>
+  <Param name="appid" value="dj00aiZpPXVyZmc2WDgzWnA5SSZzPWNvbnN1bWVyc2VjcmV0Jng9MTE-"/>
+  <Param name="fr" value="brave-mobile_ext"/>
+</Url>
+<Url type="text/html" method="GET" template="https://search.yahoo.co.jp/search/">
+  <Param name="p" value="{searchTerms}"/>
+  <Param name="ei" value="UTF-8"/>
+  <Param name="fr" value="brave-mobile_ext"/>
```

Comment:
Is it required to expose that it's `Brave`?
cc: @StephenHeaps for privacy team referral.

Reply:
his was approved via https://github.com/brave/reviews/issues/1847#issuecomment-2611308722